fastly/fastly.go

```diff
@@ -5,6 +5,17 @@ import (
 	"encoding"
 )
 
+type BatchOperation string
+
+const (
+	Create BatchOperation = "create"
+	Update BatchOperation = "update"
+	Upsert BatchOperation = "upsert"
+	Delete BatchOperation = "delete"
```

Comment:
Am slightly concerned about having `fastly` package level constants that are very generic, `Create`, `Update` etc which could cause conflict in the future. Could we consider renaming to something more specific?

Reply:
That is a good point 'BatchOpsCreate' or 'BatchOperationCreate' could be used.  I think the second one is more clear.

Reply:
Appended the name of BatchOperation to the next of each constant.